lua/gk.lua

```diff
@@ -15,8 +15,50 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local fib_entries = {
+		{
+			["ip_addr"] = "187.73.32.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["ip_addr"] = "187.73.33.0",
+			["prefix_len"] = 30,
+			["action"] = gatekeeper.c.GK_DROP,
+		},
+	}
+
+	local grantors = {
```

Comment:
I still don't see what the network admin is gaining having to keep both lists. For example, an admin adds a grantor server in this list, but it's never referred by a FIB entry. This grantor server is just taking space. The code that process the FIB entry should deduplicate whatever it can.